tools/installer/BUILD.gn

```diff
@@ -0,0 +1,119 @@
+# Copyright (c) 2016 Intel Corporation. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+# Simplified version based on //chrome/installer/linux
+import("//xwalk/build/version.gni")
+
+action("xwalk_installer_package_deb") {
+  packaging_files_binaries = [ "$root_out_dir/xwalk" ]
+  if (is_component_ffmpeg) {
+    packaging_files_binaries += [ "$root_out_dir/libffmpeg.so" ]
+  }
+
+  script = "//chrome/installer/linux/flock_make_package.py"
+  if (current_cpu == "x86") {
+    # The shell scripts use "ia32" instead of "x86".
+    build_script_arch = "ia32"
+  } else {
+    build_script_arch = current_cpu
+  }
+
+  if (current_cpu == "x86") {
+    deb_arch = "i386"
+  } else if (current_cpu == "x64") {
+    deb_arch = "amd64"
+  } else if (current_cpu == "arm") {
+    deb_arch = "arm"
+  } else {
+    assert(false, "Linux installer not configured for this architecture.")
+  }
+
+  inputs = packaging_files_binaries
+  outputs = [
+    "$root_out_dir/crosswalk-${xwalk_version}-1_${deb_arch}.deb",
+  ]
+
+  args = [
+    rebase_path("$root_out_dir/linux_package.lock", root_build_dir),
+    rebase_path("$root_out_dir/installer/debian/build.sh", root_build_dir),
+    "-o",
+    rebase_path(root_out_dir, root_build_dir),
+    "-b",
+    rebase_path(root_out_dir, root_build_dir),
+    "-a",
+    build_script_arch,
+  ]
+  deps = [
+    ":common_packaging_files",
+    ":deb_packaging_files",
+    ":xwalk_installer_linux_prepare",
+    "//xwalk",
+    "//xwalk:generate_upstream_blink_version",
+  ]
+
+  if (is_component_ffmpeg) {
```

Comment:
it is `deps +=` in below block, so I moved the whole `deps = [...]` block on top after merging `is_component_ffmpeg`